package.json

```diff
@@ -1,54 +1,85 @@
-  "repository": "https://github.com/adopted-ember-addons/active-model-adapter",
-  "description": "Adapters and Serializers for Rails's ActiveModel::Serializers",
+  "description": "The default blueprint for ember-cli addons.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": "",
+  "license": "MIT",
+  "author": "",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
   "scripts": {
-    "start": "ember server",
-    "build": "ember build",
-    "test": "ember try:testall"
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint:hbs": "ember-template-lint .",
+    "lint:js": "eslint .",
+    "lint:ts": "ember ts:precompile",
+    "start": "ember serve",
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
+    "prepublishOnly": "ember ts:precompile",
+    "postpublish": "ember ts:clean"
   },
-  "engines": {
-    "node": "10.* || 12.* || >= 14"
+  "dependencies": {
+    "ember-cli-babel": "^7.20.5",
+    "ember-cli-htmlbars": "^5.1.2",
+    "ember-cli-typescript": "^3.1.4",
+    "ember-inflector": "^4.0.1"
   },
-  "author": "",
-  "license": "MIT",
   "devDependencies": {
-    "broccoli-babel-transpiler": "^5.5.0",
-    "broccoli-concat": "0.0.13",
-    "broccoli-es3-safe-recast": "^2.0.0",
-    "broccoli-funnel": "^1.0.1",
-    "broccoli-merge-trees": "^1.1.0",
-    "broccoli-replace": "^0.11.0",
-    "broccoli-stew": "^1.0.4",
-    "ember-cli": "~2.16.2",
-    "ember-cli-dependency-checker": "^1.2.0",
-    "ember-cli-htmlbars": "^1.0.1",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.2",
-    "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-pretender": "0.5.0",
-    "ember-cli-qunit": "^4.0.1",
-    "ember-cli-shims": "^1.1.0",
-    "ember-data": "~2.6.0",
-    "ember-disable-prototype-extensions": "^1.0.1",
-    "ember-disable-proxy-controllers": "^1.0.1",
-    "ember-export-application-global": "^1.0.5",
-    "ember-load-initializers": "^1.0.0",
-    "ember-resolver": "^2.0.3",
-    "github": "^0.2.4",
-    "loader.js": "4.6.0",
-    "rsvp": "^3.1.0"
+    "@ember/optional-features": "^1.3.0",
+    "@glimmer/component": "^1.0.0",
+    "@glimmer/tracking": "^1.0.0",
+    "@types/ember": "^3.1.2",
+    "@types/ember-data": "^3.1.11",
+    "@types/ember-qunit": "^3.4.8",
+    "@types/ember__test-helpers": "^0.7.12",
+    "@types/qunit": "^2.9.1",
+    "@types/rsvp": "^4.0.3",
+    "@typescript-eslint/eslint-plugin": "^3.1.0",
+    "@typescript-eslint/parser": "^3.1.0",
+    "babel-eslint": "^10.1.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-auto-import": "^1.5.3",
+    "ember-cli": "^3.18.0",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-typescript-blueprints": "^3.0.0",
+    "ember-cli-uglify": "^3.0.0",
+    "ember-data": "^3.19.0",
```

Comment:
@igorT @runspired I believe this should be either a dependency or peer dependency, yes?

Reply:
It's dangerous to make it a dependency (consuming app then has two versions installed and may not get the right version). Last I checked peerDependencies aren't handled well anywhere in the JS ecosystem either. Most addons leave it as a devDependency due to these things.